src/quantum/azext_quantum/tests/latest/recordings/test_jobs.yaml

```diff
@@ -13,24 +13,24 @@ interactions:
+      string: '{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/e2e-scenarios/providers/Microsoft.Quantum/Workspaces/e2e-qsharp-tests","name":"e2e-qsharp-tests","type":"microsoft.quantum/workspaces","location":"westus2","tags":{},"systemData":{"createdBy":"anpaz@microsoft.com","createdByType":"User","createdAt":"2021-04-13T19:10:58.0366776Z","lastModifiedBy":"kikomiss@microsoft.com","lastModifiedByType":"User","lastModifiedAt":"2023-11-27T05:47:01.5309605Z"},"identity":{"principalId":"0dbdbf52-26d4-470c-9234-bfa298dcda68","tenantId":"72f988bf-86f1-41af-91ab-2d7cd011db47","type":"SystemAssigned"},"properties":{"providers":[{"providerId":"qci","providerSku":"qci-freepreview","applicationName":"e2e-qsharp-tests-qci","provisioningState":"Failed"},{"providerId":"rigetti","providerSku":"azure-quantum-credits","applicationName":"e2e-qsharp-tests-rigetti","provisioningState":"Failed"},{"providerId":"ionq","providerSku":"pay-as-you-go-cred","applicationName":"e2e-qsharp-tests-ionq","provisioningState":"Failed"},{"providerId":"microsoft-qc","providerSku":"learn-and-develop","applicationName":"e2e-qsharp-tests-microsoft-qc","provisioningState":"Succeeded"},{"providerId":"pasqal","providerSku":"pasqal-aq-credits","applicationName":"e2e-qsharp-tests-pasqal","provisioningState":"Succeeded","resourceUsageId":"f838a755-9e3c-4062-9b16-62ac54c386a8"},{"providerId":"quantinuum","providerSku":"credits1","applicationName":"e2e-qsharp-tests-quantinuum","provisioningState":"Failed"}],"provisioningState":"Failed","usable":"Partial","storageAccount":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/e2e-scenarios/providers/Microsoft.Storage/storageAccounts/e2etests","endpointUri":"https://e2e-qsharp-tests.westus2.quantum.azure.com"}}'
```

Comment:
So this new recording has "provisioningState":"Failed". It doesn't seem to be ideal